src/main/java/com/artipie/asto/Key.java

```diff
@@ -42,6 +43,13 @@
      */
     String string();
 
+    /**
+     * Creates child key with given name.
+     * @param name Child name.
+     * @return Child key.
+     */
+    Key child(String name);
```

Comment:
`Key.From(base, name)` could be used to do the same thing.

Reply:
Sorry, missed this constructor. Still, I believe that introducing this new child() method makes it more intuitive to create child keys from root node. I might be wrong, so it's up to you to accept it or not.